components/brave_sync/client/bookmark_change_processor.cc

```diff
@@ -935,15 +981,18 @@ void BookmarkChangeProcessor::SendUnsynced(
 
       std::string last_send_time;
       node->GetMetaInfo("last_send_time", &last_send_time);
+      size_t current_retry_number = GetCurrentRetryNumber(node);
       if (!last_send_time.empty() &&
           // don't send more often than unsynced_send_interval_
```

Comment:
we need a new comment as well

Reply:
Fixed in https://github.com/brave/brave-core/commit/6ef48bbfe9d410087d0baf342e6bd641cbcd3c9b#diff-337b5454dfaa4fc420e81d43c8bb06e3R1004